lib/templates/deploy.batch.aspnet.5.template

```diff
@@ -16,16 +16,15 @@ IF "%DEPLOYMENT_TARGET:~-1%"=="\" (
     SET DEPLOYMENT_TARGET=%DEPLOYMENT_TARGET:~0,-1%
 )
 
-IF NOT DEFINED SCM_DNX_VERSION (
```

Comment:
Why is default handling for this `ASPNET5_VERSION` (and the two new ones) not needed? Won't that cause problem when script is run directly from command line?